packages/e/eabase/xmake.lua

```diff
@@ -0,0 +1,29 @@
+package("eabase")
+
+    set_homepage("https://github.com/electronicarts/EABase")
+    set_description("EABase is a small set of header files that define platform-independent data types and platform feature macros.")
+    set_license("BSD-3-Clause")
+
+    add_urls("https://github.com/electronicarts/EABase/archive/$(version).tar.gz")
+    add_versions("2.09.06", "981f922441617152b841585c0fc0bd205bd898c758016fa4985599a63c5a6e16")
+
+    add_deps("cmake")
+
+    on_install("windows", "linux", "macosx", function (package)
+        import("package.tools.cmake").install(package)
+    end)
+
+    on_test(function (package)
+        assert(package:check_cxxsnippets({test = [[
+            template <typename T>
+	        struct eabase_template
+	        {
+		        T value;
+		        T GetValue() const { return value; }
+	        };
+	        EA_EXTERN_TEMPLATE(struct eabase_template<char>);
+            
```

Comment:
Still wrong indentation, please use spaces instead of tab

```lua
assert(package:check_cxxsnippets({test = [[
    template <typename T>
    struct eabase_template
    {
        T value;
        T GetValue() const { return value; }
    };
    EA_EXTERN_TEMPLATE(struct eabase_template<char>);

    void test() {
    }
]]}, {configs = {languages = "c++17"}}, {includes = "EABase/eabase.h"}))

```